src/keys/bip39.rs

```diff
@@ -36,33 +38,33 @@ pub enum Error {
 }
 
 /// Reference to a normalized (unicode NFKD) mnemonic.
-#[derive(Clone, Copy)]
-pub struct MnemonicRef<'a>(&'a str);
+pub struct MnemonicRef(str);
 
-impl<'a> Deref for MnemonicRef<'a> {
+impl Deref for MnemonicRef {
     type Target = str;
     fn deref(&self) -> &str {
-        self.0
+        unsafe { core::mem::transmute(self) }
```

Comment:
I think in the Rust ecosystem it's good practice to accompany an `unsafe` with a  `//Safety: <why is it still safe>` doc.

Reply:
```rust
// Safety: I have no idea what I am doing
```

Reply:
Added comment.